xtask/src/commands/schemas.rs

```diff
@@ -20,14 +19,8 @@ pub async fn handle_command(args: Args) -> Result<()> {
     // Define which types should be used to generate schemas
     let schemas = Vec::from([
         schema_for!(ClientMessage),
-        schema_for!(NewRequest),
-        schema_for!(NewRequestError),
-        schema_for!(Request),
-        schema_for!(RequestAdded),
-        schema_for!(RequestSummary),
-        schema_for!(RequestWithResponse),
-        schema_for!(Response),
         schema_for!(ServerMessage),
+        // schema_for!(fpx_lib::api::models::Span),
```

Comment:
Probably a good idea to re-run the task so the old types are removed as well

Reply:
Applied in `bd5ee9467bfe5ee7e2733cbfb48fe8bff8dc0261`